mlflow/server/js/src/experiment-tracking/components/CompareRunView.css

```diff
@@ -19,16 +32,33 @@
   overflow-x: auto;
 }
 
-.compare-table tr.row-changed th, .compare-table tr.row-changed td{
+.compare-table .diff-row .data-value {
   background-color: rgba(249, 237, 190, 0.5) ;
   color: #555;
 }
 
+.compare-table .diff-row .head-value {
+  background-color: rgba(249, 237, 190, 1.0) ;
+  color: #555;
+}
+
+.compare-table .diff-row:hover {
+  background-color: rgba(249, 237, 190, 1.0) ;
+  color: #555;
+}
+
 /* Overrides to make it look more like antd */
 .compare-table {
   width: 100%;
   max-width: 100%;
   margin-bottom: 20px;
+
+  /* The border-collapse and border-spacing setting is to make header column "position sticky"
+   * works correctly with the border-bottom style.
+   * https://stackoverflow.com/a/61412922/12301610
+   */
+  border-collapse: separate;
+  border-spacing: 0;
```

Comment:
What would happen if we removed these?

Reply:
I commented out these styles but didn't see the difference.

Reply:
It is a small style issue. without this some border lines looks thicker.

Reply:
But added this, in chrome (only in chrome, might be bug), there's also a bug display header column: 
![image](https://user-images.githubusercontent.com/19235986/151899962-1b58b5b4-21a8-47e9-b3b7-68e338a15426.png)

Reply:
Or we can hide header column border to avoid this issue
![image](https://user-images.githubusercontent.com/19235986/151902806-9fe4921b-b906-4e14-88d4-b71f0885d674.png)

Reply:
![image](https://user-images.githubusercontent.com/17039389/151904891-5e6588a1-ee3a-4bf4-a72f-0a3fd3cce64a.png)

Reply:
emm， interesting

Reply:
offline discussion, remove this because border line thicker is a tiny issue from customer side.